crypto-ws-client/src/common/connect_async.rs

```diff
@@ -17,6 +20,24 @@ pub async fn connect_async(
     url: &str,
     uplink_limit: Option<(NonZeroU32, std::time::Duration)>,
 ) -> Result<(Receiver<Message>, Sender<Message>), Error> {
+
+    if let Ok(proxy_env) = env::var("https_proxy") {
```

Comment:
Please also support `http_proxy`, for example, `if let Ok(proxy_env) = env::var("https_proxy").or(env::var("http_proxy"))`